endpoints/getting-started/clients/google-id-token-client.py

```diff
@@ -19,25 +19,24 @@
 
 import argparse
 
-import oauth2client.client
-import oauth2client.file
-import oauth2client.tools
+import google_auth_oauthlib.flow
```

Comment:
This seems to be accurate, but just out of curiosity, why is google.auth a thing and google_auth also a thing?